Makefile

```
@@ -55,6 +55,10 @@ $(CMDS):
 $(OPM): opm_version_flags=-ldflags "-X '$(PKG)/cmd/opm/version.gitCommit=$(GIT_COMMIT)' -X '$(PKG)/cmd/opm/version.opmVersion=$(OPM_VERSION)' -X '$(PKG)/cmd/opm/version.buildDate=$(BUILD_DATE)'"
 $(OPM):
 	$(extra_env) $(GO) build $(opm_version_flags) $(extra_flags) $(TAGS) -o $@ ./cmd/$(notdir $@)
+ifeq ($(shell go env GOARCH),arm64)
```

This will also hit Raspberry Pi, and miss those older Intel models that can run Sequoia. Did you want to check explicitly for macOS?
```suggestion
ifeq ($(shell go env GOOS),darwin)
```

---

I didn't want to test GOOS, because intel-based darwin doesn't have this requirement.  I couldn't figure out how to do a double conditional well to select only the darwin+arm64 set of targets.